lib/rack/lint.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 require 'forwardable'
-require 'uri'
+require 'uri/rfc3986_parser'
```

Comment:
Is this actually used now that we copied the implementation?

Reply:
We can drop it.